.github/workflows/build_and_push_sdk_packages.yaml

```diff
@@ -0,0 +1,53 @@
+# TO BE REMOVED BEFORE MERGING
+
+name: SDK Publishing
+
+on:
+  push:
+    tags:
+    # Not the right semver regexp, but good enough
+      - 'v[0-9]+.[0-9]+.[0-9]+-*'
+
+jobs:
+  sdk-packaging:
+    name: Package SDK
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+
+      - name: Install python
+        run: uv python install
+        working-directory: lumigator/python/mzai/sdk
+
+      - name: Package SDK
+        run: uv build
+        working-directory: lumigator/python/mzai/sdk
+
+      - name: Package schemas
+        run: uv build
+        working-directory: lumigator/python/mzai/schemas
+
+      - name: Upload packages
```

Comment:
We'll want to consider how we manage/delete old versions here. Our storage on GH is not unbounded.

Reply:
True. By default (https://docs.github.com/en/organizations/managing-organization-settings/configuring-the-retention-period-for-github-actions-artifacts-and-logs-in-your-organization) retention is 90 days long, and the packages are fairly light, but this kind of thing easily gets out of control :-/